tools/hypernode-vagrant-runner/tests/unit/hypernode_vagrant_runner/vagrant/set_up/test_write_hypernode_vagrant_configuration.py

```diff
@@ -30,7 +30,8 @@ def test_write_hypernode_vagrant_configuration_writes_configuration(self):
             ],
             box_url=HYPERNODE_VAGRANT_BOX_URLS[
                 HYPERNODE_VAGRANT_DEFAULT_PHP_VERSION
-            ]
+            ],
+            ubuntu_version='precise'
```

Comment:
That's the notation in our internal systems, here it is "xenial" and "precise" (from the local.yml) settings file. We could refactor it to have the same name everywhere but it wouldn't really matter since these are different systems. This flag will dissappear when we drop precise support in the future. We need this so we can perform unauthenticated builds of packages that are not tightly coupled with control on both ubuntu versions while we develop the new version. To summarize: hn-control requires a lot of overhead in automated test situations, we have projects that aren't tangled up in all that and this allows us to automatically test those inside a hypernode by just adding 1 small shellscript to each project.